fe2-android/app/src/androidTest/java/com/github/dedis/popstellar/pages/detail/LaoDetailPageObject.java

```diff
@@ -0,0 +1,25 @@
+package com.github.dedis.popstellar.pages.detail;
+
+import androidx.annotation.IdRes;
+
+import com.github.dedis.popstellar.R;
+
+public class LaoDetailPageObject {
```

Comment:
Why is this class neded?

Reply:
I tried to write the ElectionStartFragmentTest to work on the LaoDetailActivity, so started writing the LaoDetailActivity page object.

But it was a lot of work and I decided to forget it. I kept the class as it will be useful in the future.